lib/apply-state.js

```diff
@@ -255,6 +282,11 @@ module.exports = class ApplyState extends ReadyResource {
     return false
   }
 
+  removeable (key) {
+    if (this.system.indexers.length !== 1) return true
+    return !b4a.equals(this.system.indexers[0].key, key)
+  }
```

Comment:
nice, lets make removable on the base just fwd this one

Reply:
do we still need it on the base?

Reply:
ya keet uses it